src/utils/privileges.cpp

```diff
@@ -262,6 +270,8 @@ void drop_root() {
 	#else
 		_note("This security operation is not available on this system, ignoring");
 	#endif
+	datastore::set_home(home_dir);
```

Comment:
... here:

if (home_always_env) return std::string();
// else set it now:
datastore::set_home(home_dir);
return home_dir;